config/install/core.entity_form_display.node.alert.default.yml

```diff
@@ -1,3 +1,4 @@
+uuid: 11d0e2c6-f0f1-47a1-ad00-82ba079af7e0
```

Comment:
I guess we must avoid having uuid-ed configs in distro, please
It will create issues in upgrade path

Reply:
@podarok sorry, my bad. Fixed in https://github.com/open-y-subprojects/openy_node_alert/pull/146/commits/d4766904caa668a092eab382a4fc9a3b72c868cc